UndertaleModCli/Program.UMTLibInherited.cs

```diff
@@ -586,7 +587,7 @@ public bool LintUMTScript(string path)
         {
             CancellationTokenSource source = new CancellationTokenSource(100);
             CancellationToken token = source.Token;
-            CSharpScript.EvaluateAsync(File.ReadAllText(path), CliScriptOptions, this, typeof(IScriptInterface), token);
+            CSharpScript.EvaluateAsync(File.ReadAllText(path, Encoding.UTF8), CliScriptOptions.WithFilePath(path).WithFileEncoding(Encoding.UTF8), this, typeof(IScriptInterface), token);
```

Comment:
Why do we need to specify utf8 here? what if someone has a script file thats saved as ansi?

Reply:
I wasn't sure of this at the original time of writing, but apparently `File.ReadAllText` defaults to UTF-8 in the first place, so this is just being explicit about it. Technically we could try to handle other encodings some day, but we haven't up until this point. Although, UTF-8 should be backwards-compatible with other encodings like ASCII and subsets of ANSI.